site/content/faq/what-is-the-default-directory-that-zap-uses.md

```diff
@@ -7,35 +7,23 @@ weight: 2
 
 The default directory that ZAP uses depends on the OS.
 
-Note that you can override this using the -dir [command line](/docs/desktop/cmdline/) option.
+It can be overridden using the `-dir` [command line](/docs/desktop/cmdline/) option.
 
-##  Windows 7 / 8
+The default or home directory contains ZAP files added or modified at runtime. Including (but not limited to): config files, add-ons, input files, logs, etc. It is also the default save location, for: scripts, active scan policy files, exported contexts, etc.
```

Comment:
Saying "default" for policies it's still not correct to me, that implies that there are more locations but there's only one.
Anyway, IMHO it's unnecessary to go into much details in the FAQ, we can document the defaults or actual locations for each "file" in the corresponding user guide section.

Reply:
Your point makes sense, I'll drop the "default"

Reply:
> we can document the defaults or actual locations for each "file" in the corresponding user guide section.

Agreed, I was just trying to give a minimal explanation that people could refer to.